public/index.html

```diff
@@ -60,6 +60,7 @@
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
+    <div id="modal-root" class="modal-root"></div>
```

Comment:
accept as temporary option

Reply:
no)) it is needed for modal windows - portal